Classes/Service/DirectoryService.php

```diff
@@ -171,7 +171,7 @@ public function getUserEntries($username)
      * @return array
      * @throws Exception
      */
-    public function getGroupMembership($username)
+    public function getMemberOf($username)
```

Comment:
reading this actually both method names were wrong ;) one inspired on ldap, the other one on ad. But both are kind of wrong as they both do not clearly express that it will return the groups a user belongs to